UDPClientGUI/SiceUDPClient.cs

```diff
@@ -51,5 +60,32 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ClientCommandEvent Command)
+        {
+            Dictionary<ClientCommandEvent.Command, System.Action> handler = new Dictionary<ClientCommandEvent.Command, System.Action>();
+            handler.Add(ClientCommandEvent.Command.Connect, Connect);
+            handler.Add(ClientCommandEvent.Command.Disconnect, Disconnect);
+            handler.Add(ClientCommandEvent.Command.SendMessage, SendMessage);
+            InputMessage = Command.Message;
+            ConnectionIP = Command.ConnectionIP;
+            ConnectionPort = Command.ConnectionPort;
+            handler[Command.ClientCommand]();
```

Comment:
This is a nice shortcut, but better to split this up.

Also if you compare this to the switch statement it's almost the same... but it's missing a default.

Example: what happens if a new Command.ClientCommand was created but this dictionary wasn't updated? The key wouldn't exist... what would happen in that instance?

Reply:
fixed with switch clause